src/transformers/models/whisper/modeling_whisper.py

```diff
@@ -229,87 +225,15 @@ def compute_num_masked_span(input_length):
     return spec_aug_mask
 
 
-def _median_filter(inputs: torch.Tensor, filter_width: int) -> torch.Tensor:
```

Comment:
All functionality only related to generation are moved into `generation_whisper`